soa/types/time.h

```diff
@@ -0,0 +1,64 @@
+
+#pragma once
+
+#include<chrono>
+
+namespace Datacratic {
+
+constexpr std::chrono::hours operator ""h(unsigned long long h)
```

Comment:
Also, since these operators are standard in C++14, it might clash when we'll upgrade to a version of gcc that implements them.

I would suggest finding the first version of libstdc++ shipped with gcc that implement these operators and put an #if around it like so:

```cpp
#if GCC_VERSION < XXXXXX
     constexpr std::chrono::hours ...
#endif
```